main.py

```diff
@@ -14,6 +14,5 @@
 import classes
 
 logging.basicConfig(level=logging.INFO, format='[%(asctime)s - %(levelname)s - %(threadName)s] %(message)s', force=True)
-
```

Comment:
unnecessary.  ideally there should actually be two lines in between (pursuant to PEP8).

Reply:
fixed

Reply:
there is still a deletion here.